### PR TITLE
Enable gcc toolchain on Linux PPC LE platforms

### PR DESCRIPTION
**Pull request by vsebe**
Enable gcc toolchain in linux_ppc-64_le, linux_ppc-64_cmprssptrs_le
build specs. Enable OpenJ9 and J9 ddr flags.
Fix references to *_gcc specs in makefiles and other build resources.
Rename cmake specs for Linux PPC LE:
- linux_ppc-64_le_gcc to linux_ppc-64_le
- linux_ppc-64_cmprssptrs_le_gcc to linux_ppc-64_cmprssptrs_le
and update accordingly.
Remove linux_ppc-64_le_gcc and linux_ppc-64_cmprssptrs_le_gcc specs.

Depends on 
https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/412
https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/303
https://github.com/ibmruntimes/openj9-openjdk-jdk14/pull/53
https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/200

[skip ci]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
Related to:
- https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/412

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk8 depends ibmruntimes/openj9-openjdk-jdk8#412

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk11 depends ibmruntimes/openj9-openjdk-jdk11#303

**Comment by pshipton** [comment]
jenkins test sanity plinux jdk14 depends ibmruntimes/openj9-openjdk-jdk14#53

File: buildspecs/linux_ppc-64_cmprssptrs_le.spec [hunk 1]
```diff
@@ -129,7 +129,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="build_java8" value="true"/>
 		<flag id="build_java9" value="false"/>
 		<flag id="build_product" value="true"/>
+		<flag id="build_openj9" value="true"/>
```
Comment (pshipton) [inline buildspecs/linux_ppc-64_cmprssptrs_le.spec:132]: I think this and the next are  obsolete properties that could be removed from all specs? These values aren't set in the non-cmprssptrs spec, so I'm guessing they aren't used. Anyway, if we make changes for this, it should be done separately.

**Comment by pshipton** [comment]
jenkins compile plinuxxl jdk8 depends ibmruntimes/openj9-openjdk-jdk8#412

**Comment by pshipton** [comment]
jenkins compile plinuxxl jdk11 depends ibmruntimes/openj9-openjdk-jdk11#303

File: buildspecs/linux_ppc-64_cmprssptrs_le.spec (hunk 1 above)
Comment (vsebe) [reply buildspecs/linux_ppc-64_cmprssptrs_le.spec:132]: will clean up in a different PR

**Pull request merged**